### PR TITLE
Replay debugger fixes: Update the trigger parsing logic and add VF_AP…

### PR DESCRIPTION
**Pull request by JimSuplizio**
…EX* event processing

### What does this PR do?
There are two changes:
1) Update the parsing to correctly handle the typeref of the trigger formatting
2) Add VF_APEX_CALL_START/END constants and update the LogContext class so we can process those events.

The first two bugs below were for the logging portion of the fix, the second two bugs are for the VS Code changes. 

In the second iteration, unit tests were added, the callstack for triggers had the __sfdc* goo stripped from it and I incorporated feedback from the first iteration.

### What issues does this PR fix or reference?
@W-4629779@ @W-4615757@ @W-4652923@ @W-4660881@

### CODE REVIEW

File: packages/salesforcedx-vscode-replay-debugger/src/constants.ts [hunk 1]
```diff
@@ -18,4 +18,6 @@ export const EVENT_METHOD_EXIT = 'METHOD_EXIT';
 export const EVENT_STATEMENT_EXECUTE = 'STATEMENT_EXECUTE';
 export const EXEC_ANON_SIGNATURE = 'execute_anonymous_apex';
 export const GET_LINE_BREAKPOINT_INFO_EVENT = 'getLineBreakpointInfo';
-export const LINE_BREAKPOINT_INFO_REQUEST = 'lineBreakpointInfo';
\ No newline at end of file
+export const LINE_BREAKPOINT_INFO_REQUEST = 'lineBreakpointInfo';
+export const VF_APEX_CALL_START = 'VF_APEX_CALL_START';
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-replay-debugger/src/constants.ts]: Group this with the other events above and use the same naming convention.

File: packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts [hunk 2]
```diff
@@ -113,7 +115,14 @@ export class LogContext {
     const typerefMapping = this.breakpointUtil.getTyperefMapping();
     let uri = '';
     typerefMapping.forEach((value, key) => {
-      const processedKey = key.replace('/', '.').replace('$', '.');
+
+      let processedKey = "";
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts]: Use single quotes.

**Comment by DatGuyJonathan** [comment]
What does the callstack look like?

File: packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts (hunk 2 above)
Comment (JimSuplizio) [reply packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts]: Doh! I fixed this locally, I'll get a new update shortly.

File: packages/salesforcedx-vscode-replay-debugger/src/constants.ts (hunk 1 above)
Comment (JimSuplizio) [reply packages/salesforcedx-vscode-replay-debugger/src/constants.ts]: Renamed them to EVENT_VF_APEX_CALL_START and EVENT_VF_APEX_CALL_END and grouped them with the other events. I'll update this review shortly with the changes.

**Comment by DatGuyJonathan** [comment]
@JimSuplizio , here are the recommended VS Code extensions for working with this project: https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/.vscode/extensions.json

They can help detect errors like the alphabetized imports. VS Code usually asks for you to install these, but you can trigger the install manually described in https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions.

**Comment by vazexqi** [comment]
Yeah, please install the recommended extensions. The [tslint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) extension will highlight the errors as they appear.  And if you hover over it there is a lightbulb that help fix many issues automatically.

**Pull request merged**